ddtrace/span.py

```diff
@@ -196,6 +196,8 @@ def set_tag(self, key, value=None):
         # DEV: `http.status_code` *has* to be in `meta` for metrics
         #   calculated in the trace agent
         if key == http.STATUS_CODE:
+            if 500 <= int(value) <= 599:
```

Comment:
Let's leave this change out of this PR. It's a core change and we should have test cases for it separately.